CollAction/webpack.config.js

```diff
@@ -1,32 +1,42 @@
 const webpack = require('webpack');
```

Comment:
The webpack config for webpack 4 can be lot smaller (it has a lot of sensible defaults). Check what's in master branch at the moment.